docs/general/ad-filtering/create-own-filters.md

```diff
@@ -3097,7 +3111,7 @@ For example:
 
 It will check, if the code of element contains two consecutive substrings `banner` and `text`.
 
-* **`max-length`**
+#### `max-length`
```

Comment:
@ngorskikh we should add a deprecation notice here as well.

These custom attributes violate the CSS semantics and we'd better get rid of them.